library/Booster/Pattern/Rewrite.hs

```diff
@@ -487,6 +581,7 @@ performRewrite doTracing def mLlvmLibrary mbMaxDepth cutLabels terminalLabels pa
                 (Just _, [(lbl, uId, n)]) -> RewriteFinished (Just lbl) uId n
```

Comment:
Two lines above this one, `(Nothing, _) -> RewriteStuck p` should probably become `-> RewriteTrivial p` ?

Similarly for the 3 cases `Terminal`, `Finished`, `Aborted` below, which currently use `RewriteStuck`, too?